test/server/cards/01_SOR/leaders/ChewbaccaWalkingCarpet.spec.js

```diff
@@ -0,0 +1,58 @@
+describe('Chewbacca, Walking Carpet', function() {
+    integration(function(contextRef) {
+        describe('Chewbacca, Walking Carpet\'s undeployed ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['alliance-xwing', 'liberated-slaves', 'seventh-fleet-defender', 'consular-security-force'],
+                        leader: 'chewbacca#walking-carpet',
+                        base: 'administrators-tower'
```

Comment:
Can we also put some <= 3 cost cards in discard and also some smuggle-able cards in resources to ensure that his ability doesn't become able to target them whenever we add those options.

If you want you can set that up as a separate describe since unfortunately if you want to name any card in the resource pile you need to name _all_ cards in the resource pile.